src/analysis/inner_product_space/adjoint.lean

```diff
@@ -100,8 +100,12 @@ linear_isometry_equiv.of_surjective
   ..adjoint_aux }
 (λ A, ⟨adjoint_aux A, adjoint_aux_adjoint_aux A⟩)
 
+
 localized "postfix (name := adjoint) `†`:1000 := continuous_linear_map.adjoint" in inner_product
 
+@[simp] lemma adjoint_apply (A : E →L[𝕜] F) (x : F) :
+  A† x = ((to_dual 𝕜 E).symm : (normed_space.dual 𝕜 E) → E) ((to_sesq_form A) x) := rfl
```

Comment:
```suggestion
lemma adjoint_apply (A : E →L[𝕜] F) (x : F) :
  A† x = ((to_dual 𝕜 E).symm : (normed_space.dual 𝕜 E) → E) ((to_sesq_form A) x) := rfl
```
This should definitely not be a simp lemma. Since simp rewrites from left to right, it means that it would systematically transform `A† x` into `((to_dual 𝕜 E).symm : (normed_space.dual 𝕜 E) → E) ((to_sesq_form A) x)`, but of course 99% of the time the form you would actually want to see is the left-hand side...